### PR TITLE
I added image contrast to the Monitor class

### PR DESCRIPTION
**Pull request by klwlau**
I added image contrast to the Monitor class.
The function is tested on a win10 PC with duo monitor setup and works fine.
However, I did not implemented the error Raise in `contrast()`

### CODE REVIEW

File: monitorcontrol/monitor_control.py [hunk 1]
```diff
@@ -164,13 +164,43 @@ def luminance(self, value: int):
...
+
+    @property
+    def contrast(self) -> int:
+        """
+        Gets the monitors contrast.
+
+        Returns:
+            current contrast value
+
+        Raises: ##### have not implemented or checked
```
Comment (newAM) [inline monitorcontrol/monitor_control.py]:
```suggestion
        Raises:
```

File: monitorcontrol/monitor_control.py [hunk 2]
```diff
@@ -164,13 +164,43 @@ def luminance(self, value: int):
...
 
-        Raises:
+        Raises:##### have not implemented or checked
             ValueError: luminance outside of valid range
             VCPError: failed to set luminance in the VCP
         """
         code = vcp.get_vcp_code_definition("image_luminance")
         self._set_vcp_feature(code, value)
 
+
```
Comment (newAM) [inline monitorcontrol/monitor_control.py]:
```suggestion
```

File: monitorcontrol/monitor_control.py [hunk 3]
```diff
@@ -164,13 +164,43 @@ def luminance(self, value: int):
         Args:
             value: new luminance value (typically 0-100)
 
-        Raises:
+        Raises:##### have not implemented or checked
```
Comment (newAM) [inline monitorcontrol/monitor_control.py]:
```suggestion
        Raises:
```

File: monitorcontrol/monitor_control.py [hunk 4]
```diff
@@ -164,13 +164,43 @@ def luminance(self, value: int):
...
+        Raises: ##### have not implemented or checked
+            VCPError: failed to get contrast from the VCP
+        """
+        code = vcp.get_vcp_code_definition("image_contrast")
+        return self._get_vcp_feature(code)
+
+    @contrast.setter
+    def contrast(self, value: int):
+        """
+        Sets the monitors back-light contrast.
```
Comment (newAM) [inline monitorcontrol/monitor_control.py]: Is contrast tied to the back light?  I have not read the spec on this.

**Comment by newAM** [comment]
Tried it on Linux, looks good to me!  Just needs some minor changes to make sure flake8 passes and documentation can auto-generate.

File: monitorcontrol/monitor_control.py (hunk 4 above)
Comment (klwlau) [reply monitorcontrol/monitor_control.py]: It is not contrasts and brightness are two parameters in the protocol

**Comment by newAM** [comment]
Just wanted to let you know I have not forgotten about this - I am on an extended vacation right now, I'll get back to this ASAP when I can access a desktop.

**Comment by newAM** [comment]
Sorry about that - I did end up forgetting about this :(
Merging it in now!

**Pull request merged**